### ISSUE TITLE
Options not reachable in some cases

### ISSUE THREAD

**Issue by brian-man** [CryptoBlades/cryptoblades#564]

**Comment by byyyyyyyyy** [comment]
no one doing this yet right? can I go for this ticket?

**Comment by seiyria** [comment]
go for it

**Comment by byyyyyyyyy** [comment]
I also don't understand the message. what does it mean by "You have hidden the wallet warning and it would now be displayed.". In my opinion, I think what we can do is just change the message? coz I think it's reasonable why the option button is not working. coz the wallet is not connected right? correct me if I'm not getting the the toast message right.

**Comment by seiyria** [comment]
the toast message content is not the problem here. the problem is as stated in the OP.

**Comment by byyyyyyyyy** [comment]
ok copy. that was just my suggestion. so we can minimize code changes. we could just update the message and not tell the user to disable the option. it's not just the option page that's loading a blank page. all of the links. or the option page was an exception? it should still be accessed by the user?

**Comment by seiyria** [comment]
Even if we update the message, the options page and the rest should still be visible. The problem honestly is that this option exists in the first place, because we don't want users to hide it. I'm not really sure if it makes sense to keep it around, so this will probably require some thought.

**Comment by byyyyyyyyy** [comment]
yeah. that's what I thought also. the warning initially should always show if the wallet is not connected. the message is quite important rather than just showing the users a blank page. anyways, I'll keep in touch with this ticket. you can give it some thoughts first. thanks @seiyria

**Comment by newb23** [comment]
So, this option is not technically tied to the options page not being visible, the issue lies in how the rendering is happening from the App.vue. Because a wallet is not connected, the App.vue load fails, and nothing "underneath" it fails to load as well, in the is case, the options page. This "option" to hide the wallet warning was designed to allow for non-wallet pages to be accessible when no wallet was connected, but as the warning was implemented in App.vue, not the individual pages, it affects the whole site.

TL;DR - App.vue wallet check logic is what's failing. Wallet check needs to be migrated to each "wallet required" page, so that App.vue can load without error.
